src/scipp/html/formatting_datagroup_html.py

```diff
@@ -16,10 +15,13 @@
     load_dg_detail_list_tpl, load_dg_repr_tpl, load_dg_style
 
 
-def _format_shape(var: Union[Variable, DataArray, Dataset, DataGroup]) -> str:
+def _format_shape(var: Union[Variable, DataArray, Dataset, DataGroup], br_at=15) -> str:
```

Comment:
15 seems small, is that all we can fit? This is not even enough for common *single* dimensional cases. For example, `'detector_id: 10000000'` has length 21, not counting `(` or `)`.

I feel the minimum should be set such that it can at least accommodate common 1-D cases.

Reply:
Yes, I agree. Then I'll increase it to `30` instead and increase the max-width a little bit.